glean-core/ios/sdk_generator.sh

```diff
@@ -142,7 +142,7 @@ VENVDIR="${SOURCE_ROOT}/.venv"
 # We need at least pip 20.3 for Big Sur support, see https://pip.pypa.io/en/stable/news/#id48
 # Latest pip is 21.0.1
 "${VENVDIR}"/bin/pip install "pip>=20.3"
-"${VENVDIR}"/bin/pip install --upgrade glean_parser==$GLEAN_PARSER_VERSION
+"${VENVDIR}"/bin/pip install --upgrade "git+ssh://git@github.com/mozilla/glean_parser@missing-event-imports#egg=glean_parser"
```

Comment:
o_O  What's `@missing-event-imports`?

Reply:
The branch! Will go away once I do another release and we use the version number again.